client/packages/common/src/authentication/api/operations.graphql

```diff
@@ -184,3 +184,40 @@ query lastSuccessfulUserSync {
     ...UpdateUser
   }
 }
+
+query preferences($storeId: String!) {
```

Comment:
Standard query for preferences now...

Reply:
I agree fine for now... 

This critique is true of most of our app's API consumption but particular here I believe we make a query for each preference specified so every time we call this we do a good 30 DB queries to get the response. (I haven't looked, maybe there is some loader shenanigans where we query once for all of them into a hashmap?)

Would be nice to have something like `usePreferences(storeId: String, keys: String[])` that makes the actually appropriate gql query. As it is across the app we pay for the complexity of implementing gql and then don't bother actually making specific gql queries 😅.